docs/chat/chat-schema.yaml

```diff
@@ -0,0 +1,41 @@
+definitions:
```

Comment:
Add chatBody that represents a body of createChat request

Reply:
done